### DISCUSSION TITLE
Unable to install filamentpanel

### DISCUSSION THREAD

**Discussion by polibee** [filamentphp/filament#11929] [Help]
### Package

Panel builder

### Package Version

V3.2

### How can we help you?

I tried laragon and had the same problem. I didn't have this problem with the installation panel before, and today the project download panel for redeploying laravel has this problem.
 Runtime Log
D:\phpEnv\www\app>composer require filament/filament:"^3.2" -W
Composer could not detect the root package (laravel/laravel) version, defaulting to '1.0.0'. See https://getcomposer.org/root-version
./composer.json has been updated
Composer could not detect the root package (laravel/laravel) version, defaulting to '1.0.0'. See https://getcomposer.org/root-version
Running composer update filament/filament --with-all-dependencies
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - filament/filament v3.2.0 requires filament/actions v3.2.0 -> satisfiable by filament/actions[v3.2.0].
    - filament/filament v3.2.1 requires filament/actions v3.2.1 -> satisfiable by filament/actions[v3.2.1].
    - filament/filament v3.2.2 requires filament/actions v3.2.2 -> satisfiable by filament/actions[v3.2.2].
    - filament/filament v3.2.3 requires filament/actions v3.2.3 -> satisfiable by filament/actions[v3.2.3].
    - filament/filament v3.2.4 requires filament/actions v3.2.4 -> satisfiable by filament/actions[v3.2.4].
    - filament/filament v3.2.5 requires filament/actions v3.2.5 -> satisfiable by filament/actions[v3.2.5].
    - filament/filament v3.2.6 requires filament/actions v3.2.6 -> satisfiable by filament/actions[v3.2.6].
    - filament/filament v3.2.7 requires filament/actions v3.2.7 -> satisfiable by filament/actions[v3.2.7].
    - filament/filament v3.2.8 requires filament/actions v3.2.8 -> satisfiable by filament/actions[v3.2.8].
    - filament/filament v3.2.9 requires filament/actions v3.2.9 -> satisfiable by filament/actions[v3.2.9].
    - filament/filament v3.2.10 requires filament/actions v3.2.10 -> satisfiable by filament/actions[v3.2.10].
    - filament/filament v3.2.11 requires filament/actions v3.2.11 -> satisfiable by filament/actions[v3.2.11].
    - filament/filament v3.2.12 requires filament/actions v3.2.12 -> satisfiable by filament/actions[v3.2.12].
    - filament/filament v3.2.13 requires filament/actions v3.2.13 -> satisfiable by filament/actions[v3.2.13].
    - filament/filament v3.2.14 requires filament/actions v3.2.14 -> satisfiable by filament/actions[v3.2.14].
    - filament/filament v3.2.15 requires filament/actions v3.2.15 -> satisfiable by filament/actions[v3.2.15].
    - filament/filament v3.2.16 requires filament/actions v3.2.16 -> satisfiable by filament/actions[v3.2.16].
    - filament/filament v3.2.17 requires filament/actions v3.2.17 -> satisfiable by filament/actions[v3.2.17].
    - filament/filament v3.2.18 requires filament/actions v3.2.18 -> satisfiable by filament/actions[v3.2.18].
    - filament/filament v3.2.19 requires filament/actions v3.2.19 -> satisfiable by filament/actions[v3.2.19].
    - filament/filament v3.2.20 requires filament/actions v3.2.20 -> satisfiable by filament/actions[v3.2.20].
    - filament/filament v3.2.21 requires filament/actions v3.2.21 -> satisfiable by filament/actions[v3.2.21].
    - filament/filament v3.2.22 requires filament/actions v3.2.22 -> satisfiable by filament/actions[v3.2.22].
    - filament/filament v3.2.23 requires filament/actions v3.2.23 -> satisfiable by filament/actions[v3.2.23].
    - filament/filament v3.2.24 requires filament/actions v3.2.24 -> satisfiable by filament/actions[v3.2.24].
    - filament/filament v3.2.25 requires filament/actions v3.2.25 -> satisfiable by filament/actions[v3.2.25].
    - filament/filament v3.2.26 requires filament/actions v3.2.26 -> satisfiable by filament/actions[v3.2.26].
    - filament/filament v3.2.27 requires filament/actions v3.2.27 -> satisfiable by filament/actions[v3.2.27].
    - filament/filament v3.2.28 requires filament/actions v3.2.28 -> satisfiable by filament/actions[v3.2.28].
    - filament/filament v3.2.29 requires filament/actions v3.2.29 -> satisfiable by filament/actions[v3.2.29].
    - filament/filament v3.2.30 requires filament/actions v3.2.30 -> satisfiable by filament/actions[v3.2.30].
    - filament/filament v3.2.31 requires filament/actions v3.2.31 -> satisfiable by filament/actions[v3.2.31].
    - filament/filament v3.2.32 requires filament/actions v3.2.32 -> satisfiable by filament/actions[v3.2.32].
    - filament/filament v3.2.33 requires filament/actions v3.2.33 -> satisfiable by filament/actions[v3.2.33].
    - filament/filament v3.2.34 requires filament/actions v3.2.34 -> satisfiable by filament/actions[v3.2.34].
    - filament/filament v3.2.35 requires filament/actions v3.2.35 -> satisfiable by filament/actions[v3.2.35].
    - filament/filament v3.2.36 requires filament/actions v3.2.36 -> satisfiable by filament/actions[v3.2.36].
    - filament/filament v3.2.37 requires filament/actions v3.2.37 -> satisfiable by filament/actions[v3.2.37].
    - filament/filament v3.2.38 requires filament/actions v3.2.38 -> satisfiable by filament/actions[v3.2.38].
    - filament/filament v3.2.39 requires filament/actions v3.2.39 -> satisfiable by filament/actions[v3.2.39].
    - filament/filament v3.2.40 requires filament/actions v3.2.40 -> satisfiable by filament/actions[v3.2.40].
    - filament/filament v3.2.41 requires filament/actions v3.2.41 -> satisfiable by filament/actions[v3.2.41].
    - filament/filament v3.2.42 requires filament/actions v3.2.42 -> satisfiable by filament/actions[v3.2.42].
    - filament/filament v3.2.43 requires filament/actions v3.2.43 -> satisfiable by filament/actions[v3.2.43].
    - filament/filament v3.2.44 requires filament/actions v3.2.44 -> satisfiable by filament/actions[v3.2.44].
    - filament/filament v3.2.45 requires filament/actions v3.2.45 -> satisfiable by filament/actions[v3.2.45].
    - filament/filament v3.2.46 requires filament/actions v3.2.46 -> satisfiable by filament/actions[v3.2.46].
    - filament/filament v3.2.47 requires filament/actions v3.2.47 -> satisfiable by filament/actions[v3.2.47].
    - filament/filament v3.2.48 requires filament/actions v3.2.48 -> satisfiable by filament/actions[v3.2.48].
    - filament/filament v3.2.49 requires filament/actions v3.2.49 -> satisfiable by filament/actions[v3.2.49].
    - filament/filament v3.2.50 requires filament/actions v3.2.50 -> satisfiable by filament/actions[v3.2.50].
    - filament/filament v3.2.51 requires filament/actions v3.2.51 -> satisfiable by filament/actions[v3.2.51].
    - filament/filament v3.2.52 requires filament/actions v3.2.52 -> satisfiable by filament/actions[v3.2.52].
    - filament/actions[v3.2.0, ..., v3.2.52] require openspout/openspout ^4.23 -> found openspout/openspout[4.x-dev] but it does not match your minimum-stability.
    - Root composer.json requires filament/filament ^3.2 -> satisfiable by filament/filament[v3.2.0, ..., v3.2.52].

Installation failed, reverting ./composer.json and ./composer.lock to their original content.

**Comment by leandrocfe** [comment]
please share your composer.json

**Reply by polibee** [reply]
Thanks for your help, this issue has been resolved. The official documentation delete some detail( "minimum-stability": "dev",).
